src/components/WeeklySummary/WeeklySummary.jsx

```diff
@@ -46,6 +52,25 @@ export class WeeklySummary extends Component {
       weeklySummariesCount: 0,
       mediaConfirm: false,
     },
+    uploadDate: moment()
+      .tz('America/Los_Angeles')
+      .endOf('week')
+      .toISOString(),
+    uploadDateLastWeek: moment()
+      .tz('America/Los_Angeles')
+      .endOf('week')
+      .subtract(1, 'week')
+      .toISOString(),
+    uploadDateBeforeLast: moment()
+      .tz('America/Los_Angeles')
+      .endOf('week')
+      .subtract(2, 'week')
+      .toISOString(),
+    uploadDateThreeWeeksAgo: moment()
+      .tz('America/Los_Angeles')
+      .endOf('week')
+      .subtract(3, 'week')
+      .toISOString(),
```

Comment:
Duplicate code could be replaced with a function. (eg. uploadDateFromXWeeksAgo(0-3))